src/views/identity/user/modules/AddNotificationData.vue

```diff
@@ -3,23 +3,23 @@
         <p-field-group
             :label="$t('IDENTITY.USER.NOTIFICATION.FORM.CHANNEL_NAME')"
             required
+            :invalid-text="nameInvalidText"
+            :invalid="isNameInvalid"
             class="base-info-input"
         >
             <template #default="{invalid}">
```

Comment:
slot props 는 없어도 될 것 같습니당
```suggestion
            <template #default>
```

Reply:
넵넵! 수정하겠숩니당 ㅎㅎ